pypif_sdk/func/calculate_funcs.py

```diff
@@ -0,0 +1,258 @@
+import pypif
+import re
+from mendeleev import element
+from pypif.obj import *
+
+def calculate_ideal_atomic_percent(pif):
+    """
+    Calculates ideal atomic percents from a chemical formula string from a pif. Returns an appended pif with composition elements modified or added.
```

Comment:
The other methods use `:param :` annotations; any reason not to include them here too?